supertokens_python/recipe/thirdpartypasswordless/utils.py

```diff
@@ -44,6 +57,13 @@ def __init__(self,
         self.create_and_send_custom_email = create_and_send_custom_email
 
 
+class InputPasswordlessConfig:
+    def __init__(self,
+                 create_and_send_custom_email: Union[
+                     Callable[[TypePasswordlessEmailDeliveryInput, Dict[str, Any]], Awaitable[None]], None] = None):
+        self.create_and_send_custom_email = create_and_send_custom_email
```

Comment:
You do not need to make a separate class type for this. You can directly pass `create_and_send_custom_email` into the backwards compatibility class

Reply:
@KShivendu missed this comment

Reply:
Did as you suggested.